lwmem/src/lwmem/lwmem.c

```diff
@@ -131,15 +131,15 @@
  * \param[in]       in_p: Previous of input block
  */
 #define LWMEM_GET_PREV_CURR_OF_BLOCK(in_lw, in_b, in_pp, in_p) do {     \
-        for ((in_pp) = NULL, (in_p) = &(LWMEM_GET_LW(in_lw)->start_block);  \
+        for ((in_pp) = NULL, (in_p) = &(in_lw->start_block);  \
```

Comment:
Since this is a macro, it should be done in a way `&((in_lw)->start_block)` instead